datalad/tests/test_nonasyncrunner.py

```diff
@@ -0,0 +1,159 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil; coding: utf-8 -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Test the thread based runner (aka. non asyncio based runner).
```

Comment:
hm... codecov is bugging out? how this file could be not test covered???

Reply:
I don't know. I get 100% coverage locally when I execute the tests (on linux, python 3.6). Are we sure that this issue will still exist after merging?

Reply:
let's consider it a codecov etc fluke, sorry for the noise